suite-common/connect-init/src/__tests__/trezorConnectActions.test.ts

```diff
@@ -0,0 +1,217 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Please remove original test file in the end it doesn't make sense to have it twice

Reply:
I am not sure. There will probably be a difference in error handling - when an error occurs in `prepareConnectInit`, **suite** is catching the error and dispatch further action to deal with it (that's the reason I have added `.unwrap() ` here https://github.com/trezor/trezor-suite/pull/5760/files#diff-2acc8432632da4d879e822e9644237af62d547114f0f092524383540492071f3R47). While `prepareConnectInit` is async thunk from redux/toolkit, **suite-native** will handle _prepareConnectInit.rejected_ thunk state. And this is the difference in both tests and I would prefer not to unify them at least for now.